{"message":"Unable to ensure uniqueness for usernames: ","name":"MongoError","stack":"MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux/parse-server-example/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux/parse-server-example/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux/parse-server-example/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","level":"warn","timestamp":"2017-09-14T17:41:42.873Z"}
{"message":"Unable to ensure uniqueness for user email addresses: ","name":"MongoError","stack":"MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux/parse-server-example/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux/parse-server-example/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux/parse-server-example/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","level":"warn","timestamp":"2017-09-14T17:41:42.876Z"}
{"message":"Unable to ensure uniqueness for role name: ","name":"MongoError","stack":"MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux/parse-server-example/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux/parse-server-example/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux/parse-server-example/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","level":"warn","timestamp":"2017-09-14T17:41:42.876Z"}
{"message":"Unable to ensure uniqueness for usernames: ","name":"MongoError","stack":"MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux/parse-server-example/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux/parse-server-example/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux/parse-server-example/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","level":"warn","timestamp":"2017-09-14T17:44:19.496Z"}
{"message":"Unable to ensure uniqueness for user email addresses: ","name":"MongoError","stack":"MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux/parse-server-example/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux/parse-server-example/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux/parse-server-example/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","level":"warn","timestamp":"2017-09-14T17:44:19.544Z"}
{"message":"Unable to ensure uniqueness for role name: ","name":"MongoError","stack":"MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux/parse-server-example/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux/parse-server-example/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux/parse-server-example/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","level":"warn","timestamp":"2017-09-14T17:44:19.547Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux/parse-server-example/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux/parse-server-example/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux/parse-server-example/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }","timestamp":"2017-09-14T17:44:58.888Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux/parse-server-example/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux/parse-server-example/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux/parse-server-example/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' } MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux/parse-server-example/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux/parse-server-example/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux/parse-server-example/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","timestamp":"2017-09-14T17:44:58.891Z"}
{"message":"Unable to ensure uniqueness for usernames: ","name":"MongoError","stack":"MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux/parse-server-example/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux/parse-server-example/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux/parse-server-example/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","level":"warn","timestamp":"2017-09-14T17:47:27.785Z"}
{"message":"Unable to ensure uniqueness for user email addresses: ","name":"MongoError","stack":"MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux/parse-server-example/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux/parse-server-example/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux/parse-server-example/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","level":"warn","timestamp":"2017-09-14T17:47:27.792Z"}
{"message":"Unable to ensure uniqueness for role name: ","name":"MongoError","stack":"MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux/parse-server-example/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux/parse-server-example/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux/parse-server-example/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","level":"warn","timestamp":"2017-09-14T17:47:27.796Z"}
{"functionName":"hello","params":{"score":1337,"playerName":"Sean Plott","cheatMode":false},"level":"info","message":"Ran cloud function hello for user undefined with:\n  Input: {\"score\":1337,\"playerName\":\"Sean Plott\",\"cheatMode\":false}\n  Result: \"Hi\"","timestamp":"2017-09-14T17:48:02.669Z"}
{"functionName":"hello","params":{"score":1337,"playerName":"Sean Plott","cheatMode":false},"level":"info","message":"Ran cloud function hello for user undefined with:\n  Input: {\"score\":1337,\"playerName\":\"Sean Plott\",\"cheatMode\":false}\n  Result: \"Hi\"","timestamp":"2017-09-14T17:54:04.206Z"}
{"message":"Unable to ensure uniqueness for usernames: ","name":"MongoError","stack":"MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","level":"warn","timestamp":"2017-09-14T20:50:07.282Z"}
{"message":"Unable to ensure uniqueness for user email addresses: ","name":"MongoError","stack":"MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","level":"warn","timestamp":"2017-09-14T20:50:07.306Z"}
{"message":"Unable to ensure uniqueness for role name: ","name":"MongoError","stack":"MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","level":"warn","timestamp":"2017-09-14T20:50:07.308Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }","timestamp":"2017-09-14T20:50:17.836Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' } MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","timestamp":"2017-09-14T20:50:17.839Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }","timestamp":"2017-09-14T20:50:17.865Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' } MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","timestamp":"2017-09-14T20:50:17.866Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }","timestamp":"2017-09-14T20:50:18.046Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' } MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","timestamp":"2017-09-14T20:50:18.047Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }","timestamp":"2017-09-14T20:50:18.070Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' } MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","timestamp":"2017-09-14T20:50:18.071Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }","timestamp":"2017-09-14T20:50:18.179Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' } MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","timestamp":"2017-09-14T20:50:18.180Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }","timestamp":"2017-09-14T20:50:18.589Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' } MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","timestamp":"2017-09-14T20:50:18.590Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }","timestamp":"2017-09-14T20:50:18.739Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' } MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","timestamp":"2017-09-14T20:50:18.739Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }","timestamp":"2017-09-14T20:50:18.932Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' } MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","timestamp":"2017-09-14T20:50:18.933Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }","timestamp":"2017-09-14T20:50:19.639Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' } MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","timestamp":"2017-09-14T20:50:19.640Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }","timestamp":"2017-09-14T20:50:20.748Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' } MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","timestamp":"2017-09-14T20:50:20.749Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }","timestamp":"2017-09-14T20:50:25.305Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' } MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","timestamp":"2017-09-14T20:50:25.307Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }","timestamp":"2017-09-14T20:50:25.335Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' } MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","timestamp":"2017-09-14T20:50:25.336Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }","timestamp":"2017-09-14T20:50:25.494Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' } MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","timestamp":"2017-09-14T20:50:25.495Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }","timestamp":"2017-09-14T20:50:25.523Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' } MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","timestamp":"2017-09-14T20:50:25.524Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }","timestamp":"2017-09-14T20:50:25.585Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' } MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","timestamp":"2017-09-14T20:50:25.586Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }","timestamp":"2017-09-14T20:50:25.793Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' } MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","timestamp":"2017-09-14T20:50:25.793Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }","timestamp":"2017-09-14T20:50:25.813Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' } MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","timestamp":"2017-09-14T20:50:25.815Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }","timestamp":"2017-09-14T20:50:26.040Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' } MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","timestamp":"2017-09-14T20:50:26.040Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }","timestamp":"2017-09-14T20:50:26.596Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' } MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","timestamp":"2017-09-14T20:50:26.596Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }","timestamp":"2017-09-14T20:50:27.294Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' } MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","timestamp":"2017-09-14T20:50:27.295Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }","timestamp":"2017-09-14T20:50:27.756Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' } MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","timestamp":"2017-09-14T20:50:27.757Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }","timestamp":"2017-09-14T20:50:27.880Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' } MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","timestamp":"2017-09-14T20:50:27.880Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }","timestamp":"2017-09-14T20:50:28.033Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' } MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","timestamp":"2017-09-14T20:50:28.033Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }","timestamp":"2017-09-14T20:50:28.931Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' } MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","timestamp":"2017-09-14T20:50:28.931Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }","timestamp":"2017-09-14T20:50:29.777Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' } MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","timestamp":"2017-09-14T20:50:29.778Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }","timestamp":"2017-09-14T20:50:29.925Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' } MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","timestamp":"2017-09-14T20:50:29.926Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }","timestamp":"2017-09-14T20:50:29.951Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' } MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","timestamp":"2017-09-14T20:50:29.951Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }","timestamp":"2017-09-14T20:50:30.073Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' } MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","timestamp":"2017-09-14T20:50:30.074Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }","timestamp":"2017-09-14T20:50:30.165Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' } MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","timestamp":"2017-09-14T20:50:30.166Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }","timestamp":"2017-09-14T20:50:30.562Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' } MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","timestamp":"2017-09-14T20:50:30.562Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }","timestamp":"2017-09-14T20:50:30.651Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' } MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","timestamp":"2017-09-14T20:50:30.651Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }","timestamp":"2017-09-14T20:50:30.944Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' } MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","timestamp":"2017-09-14T20:50:30.945Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }","timestamp":"2017-09-14T20:50:31.119Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' } MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","timestamp":"2017-09-14T20:50:31.121Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }","timestamp":"2017-09-14T20:50:31.305Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' } MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","timestamp":"2017-09-14T20:50:31.306Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }","timestamp":"2017-09-14T20:50:31.662Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' } MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/topologies/server.js:336:35)\n    at emitOne (events.js:115:13)\n    at Pool.emit (events.js:210:7)\n    at Connection.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/pool.js:280:12)\n    at Object.onceWrapper (events.js:318:30)\n    at emitTwo (events.js:125:13)\n    at Connection.emit (events.js:213:7)\n    at Socket.<anonymous> (/Users/kdomen/parseserver/914pelinux2/parse-server-ken/node_modules/mongodb-core/lib/connection/connection.js:187:49)\n    at Object.onceWrapper (events.js:316:30)\n    at emitOne (events.js:115:13)","timestamp":"2017-09-14T20:50:31.662Z"}
